concordium-sdk/src/main/java/com/concordium/sdk/responses/blocksummary/updates/queues/AnonymityRevokerInfo.java

```diff
@@ -28,6 +28,7 @@ public final class AnonymityRevokerInfo {
      */
     private final Description description;
 
+    @ToString.Exclude
```

Comment:
Why exclude this from the `toString()`?

Reply:
Public Key was large while trying to print the Object in Console. Hence removed it just to beautify the tostring method output. 
The assumption here is that Public Key as an Object is not useful when printed on a terminal.

Reply:
Isn't it just a hex string? 

I mean, the purpose of the `toString` is mainly debugging - and here it is usually good practice to include all the fields.

Reply:
fixed